tests/std/tests/P1147R1_printing_volatile_pointers/test.cpp

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) Microsoft Corporation.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#include <cassert>
+#include <sstream>
+
+using namespace std;
+
+template <typename T>
+string getTextValue(T* ptr) {

```

Comment:
Technically need to include `<string>` (surprisingly not guaranteed to be dragged in by `<sstream>`).